packages/shared/styles/components/organisms/SfGroupedProduct.scss

```diff
@@ -132,6 +132,11 @@
     --grouped-product-item-price-margin: 0 0 0 auto;
     .sf-grouped-product-item__quantity-selector {
       --quantity-selector-background: var(--c-light);
+      position: absolute;
```

Comment:
The same properties are in 82-86 lines. Do we need to double them? :)

Reply:
yes :), because there are two classes: one with carousel  `.sf-grouped-product-item__quantity-selector` and other without the carousel   - `.sf-grouped-product.without-carousel .sf-grouped-product-item__quantity-selector`